scripts/generate_transifex_batches.py

```
@@ -0,0 +1,339 @@
+
+def convert_locale_format(locale: str) -> str:
+    """
+    Convert internal locale format to Transifex format.
+
+    Internal: pt_BR, af_ZA (underscore for region)
+    Transifex: pt-BR, af-ZA (hyphen for region)
+    """
+    return locale.replace("_", "-")
+
+
+def generate_batches(batch_size: int, max_parallel: int, priority_based: bool = False, locales: Optional[List[str]] = None) -> List[Dict]:
```

Ruff (0.14.11)

Unused function argument: `max_parallel`

(ARG001)